### DISCUSSION TITLE
Canale Telegram, Gruppo Whatsapp o gruppo discord - [CLOSED]

### DISCUSSION THREAD

**Discussion by mancusoa74** [ittagnelli/ats#9] [General]
Al fine di avere un canale di comunicazione di facile utilizzo con i clienti ATS, proporrei di creare un canale ad hoc per gli utenti ed un canale per noi.

Dobbiamo scegliere la piattaforma pensando anche ad usi futuri ed integrazioni

- [ ] telegram [8]
- [x] whatsapp [8]
- [ ] discord [0]

Votate la vostra preferenza incrementando il contatore o aggiungete altre proposte incrementando il contatore

**Consegna: 30/9/2021**

**Comment by mancusoa74** [comment]
vedo che c'e' parità.

Quel che ci serve a noi è un canale di Telegram, in cui possiamo esserci noi amministratori ed i clienti possono scrivere.

Come si realizza un canale telegram su whatsapp?

**Reply by Alexandru-Chindris** [reply]
Si potrebbe creare un gruppo dove possono scrivere solo gli amministratori![Screenshot_20210930-190549_WhatsApp.jpg](https://user-images.githubusercontent.com/89776299/135499822-ac841a60-3cdd-43ef-94de-ea96b177beca.jpg)

**Reply by Alexandru-Chindris** [reply]
Per il gruppo "aperto a tutti" a quanto ne so si può entrare solo su aggiunta, su invito di un utente amministratore

**Reply by Alexandru-Chindris** [reply]
Su Telegram c'è la possibilità di creare un canale news(amministratori) e associargli il relativo gruppo pubblico (con pulsante diretto che collega il gruppo) . Mentre su whatsapp questa funzione non disponibile

**Reply by Dario-Ruggieri** [reply]
Sì infatti, sono d'accordo. Penso che Telegram sia utile come canale News di ATS. Simile ai gruppi per le offerte Amazon, solo che noi mandiamo le notizie di ATS. Semplicemente l'utente si deve iscrivere al canale (che possiamo diffondere con la locandina).
@mancusoa74 @Alexandru-Chindris

**Reply by Alexandru-Chindris** [reply]
@mancusoa74 
@Dario-Ruggieri @Giacomo-Lurgo 
Professore quindi vogliamo creare anche un canale Telegram o semplicemente rimanere sulla email?
